server/docker-base/Dockerfile

```diff
@@ -422,6 +422,8 @@ ENV IOS_15_VERSION          15.2
 ENV XCODE_13_CLANG_VERSION  13.0.0
 ENV SWIFT_5_5_VERSION       5.5
 
+ENV EXTENSION_PODFILE_TEMPLATE /var/extender/template.podfile
```

Comment:
We're not using a Docker container for iOS/macOS builds, but if we do in the future we need to make sure this file is available in the container.